app/run.py

```diff
@@ -51,7 +64,11 @@ def calculate_feet(formatted_data):
     :param formatted_data: dict of L/W/H information
     :return: integer for the number of feet required by performing `((Length * 2) + (Width * 2)) * Height`
     """
-    return int(formatted_data['length']) * int(formatted_data['width']) * int(formatted_data['height'])
```

Comment:
This was obviously wrong. Also I believe that input sanitation should be done here. Check for none, negative values, try cast into integer etc